File: JS-Basics/ifStatements/dayOfWeek.js
function dayOfWeek(input){
    
    let numInput = Number(input[0]);

    let outputDay;

    switch(numInput){
        case numInput = 1:
            outputDay = "Monday";
            break;
        case 2:
            outputDay = "Tuesday";
            break;
        case 3:
            outputDay = "Wednesday";
            break;
        case 4:
            outputDay = "Thursday";
            break;
        case 5:
            outputDay = "Friday";
            break;
        case 6:
            outputDay = "Saturday";
            break;
        case 7:
            outputDay = "Sunday";
            break;
        default:
            outputDay = "Error";
            break;
    }

    console.log(outputDay);

}

dayOfWeek(["1"]);

File: Java Fundamentals/Arrays/src/DayOfWeek.java
import java.util.Scanner;

public class DayOfWeek {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        String daysOfWeek[] = {"Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"};

        int inputNum = Integer.parseInt(scan.nextLine());
        if(inputNum >= 1 && inputNum <= 7){
            System.out.printf(daysOfWeek[inputNum-1]);
        }else{
            System.out.println("Invalid day!");
        }
    }
}
